spring-cloud-gcp-pubsub/src/main/java/org/springframework/cloud/gcp/pubsub/support/DefaultSubscriberFactory.java

```diff
@@ -249,4 +264,9 @@ public SubscriberStub createSubscriberStub(RetrySettings retrySettings) {
 			throw new RuntimeException("Error creating the SubscriberStub", e);
 		}
 	}
+
+	@Override
+	public PubSubAcknowledger createAcknowledger() {
+		return new DefaultPubSubAcknowledger(createSubscriberStub());
```

Comment:
What you do here can simply go intto the `default` method on the interface.

Reply:
Why do we come back to this?
What is wrong with the `default` on the interface?

Thanks

Reply:
Compiler won't let me have this here and a package-private `DefaultPubSubAcknowledger`.